src/Stratis.Bitcoin/Consensus/BlockDownloadRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using NBitcoin;
+
+namespace Stratis.Bitcoin.Consensus
+{
+    /// <summary>
+    /// A request that holds information of blocks to download.
```

Comment:
request to block puller

it holds chained headers of the blocks that are requested for download

Reply:
OK I can improve it as part of this PR too no pb